var/spack/repos/builtin/packages/povray/package.py

```diff
@@ -149,11 +149,10 @@ def configure_args(self):
         return extra_args
 
     def test(self):
+        "Render sample file"
         povs = find(self.prefix.share, "biscuit.pov")[0]
         copy(povs, ".")
-        self.run_test(
-            "povray",
-            options=["biscuit.pov"],
-            purpose="test: render sample file",
-            expected=["POV-Ray finished"],
-        )
+        exe = which("povray")
+        out = exe("biscuit.pov")
+        expected = "POV-Ray finished"
+        assert out in expected
```

Comment:
I would combine lines 157 and 158, but the ordering on line 158 will always fail since `out` is likely to contain far more than `expected`.